src/components/AnalysisModal.js

```diff
@@ -3,20 +3,31 @@ import { Fragment, useState } from 'react'
 import { div, h, h2, hr, img, span } from 'react-hyperscript-helpers'
 import { ButtonPrimary, IdContainer, Select, spinnerOverlay, WarningTitle } from 'src/components/common'
 import { ComputeModalBase } from 'src/components/ComputeModal'
+import { CromwellModalBase } from 'src/components/CromwellModal'
 import Dropzone from 'src/components/Dropzone'
 import { GalaxyModalBase } from 'src/components/GalaxyModal'
 import { icon } from 'src/components/icons'
 import ModalDrawer from 'src/components/ModalDrawer'
-import { analysisNameInput, analysisNameValidator, getDisplayName, getTool, notebookData, tools } from 'src/components/notebook-utils'
+import {
+  analysisNameInput,
+  analysisNameValidator,
+  getAppType,
+  getDisplayName,
+  getTool,
+  isToolAnApp,
+  notebookData,
+  tools
+} from 'src/components/notebook-utils'
 import TitleBar from 'src/components/TitleBar'
+import cromwellImg from 'src/images/cromwell-logo.png'
```

Comment:
We typically pre-process images by dragging and dropping them on https://tinypng.com and then downloading the resulting compressed image.

![image](https://user-images.githubusercontent.com/5438223/142513251-c9af4c25-d75a-4788-922f-a0f3a0169e5f.png)

Reply:
This is not the final image yet (Tiffany is still working on it), but I will compress the placeholder one.